ledger/src/shred.rs

```diff
@@ -198,10 +198,20 @@ enum ShredVariant {
     // the shred variant:
     //   0b0100_????  MerkleCode
     //   0b0110_????  MerkleCode chained
+    //   0b0111_????  MerkleCode chained resigned
     //   0b1000_????  MerkleData
     //   0b1001_????  MerkleData chained
-    MerkleCode { proof_size: u8, chained: bool }, // 0b01?0_????
-    MerkleData { proof_size: u8, chained: bool }, // 0b100?_????
+    //   0b1011_????  MerkleData chained resigned
+    MerkleCode {
+        proof_size: u8,
+        chained: bool,
+        resigned: bool,
```

Comment:
We only expect a single additional signature added by retransmitters, right?
E.g. if I'm in L2, I only expect signatures from the leader and the L1 node that sent to me because the L1 node strips off the signature from L0/root node.